frontend/src/views/StartView.vue

```diff
@@ -0,0 +1,217 @@
+        information may be used by the Permitting Solutions Office for survey purposes. If you have any questions
+        regarding the use of this personal information, please contact Housing Authorizations at
+        <a href="mailto:Housing.Authorizations@gov.bc.ca">Housing.Authorizations@gov.bc.ca.</a>
+      </p>
+    </div>
+
+    <h4 class="h4-title mt-2">Before you begin</h4>
+    <div class="p-disclaimer">
+      <p class="p-red">
+        Please connect with your local government for requirements on local zoning, bylaws, and permitting rules. We
+        advise that you engage with First Nations on land and resource decisions as early as possible, if applicable. Go
+        to
+        <a
+          href="https://permitconnectbc.gov.bc.ca/#engaging"
+          target="”_blank”"
+        >
+          Permit Connect BC
+        </a>
+        for more information.
+        <span class="span-underline">
+          Please be aware that the Navigators Service cannot overrule the timelines and decisions of other line of
+          businesses.
+        </span>
+      </p>
+    </div>
+    <div class="flex flex-wrap gap-3 justify-content-between mt-4">
+      <div>
+        <RadioButton
+          v-model="acceptDisclaimer"
+          value="accepted"
+          @click="flipAcceptDisclaimer"
+        />
+        <label
+          for="ingredient1"
+          class="ml-2"
+        >
+          <span
+            class="span-agree"
+            @click="flipAcceptDisclaimer"
+          >
+            <b>Yes, I understand</b>
+          </span>
+          <span class="pl-1 span-red">*</span>
+        </label>
+        <div />
+      </div>
+      <Button
+        label="Start application"
+        outlined
+        class="ml-2"
+        :disabled="!acceptDisclaimer"
+        @click="handleStartApplication"
+      />
+    </div>
+  </div>
+</template>
+
+<style scoped lang="scss">
+.p-breadcrumb {
+  border: none;
+  text-decoration: none;
+}
+
+.no-underline {
+  text-decoration: none;
+}
+
+.p-disclaimer {
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 0.5rem;
+  border-color: var(--surface-300);
+  padding: 0rem 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 0.5rem;
+}
+
+.span-agree {
+  color: var(--text-color);
+  cursor: pointer;
+}
+
+.p-red {
+  color: red;
+}
+
+.span-underline {
+  text-decoration: underline;
+}
+
+.span-red {
+  color: red;
+}
+
+.p-menuitem-link {
+  text-decoration: none !important;
+}
+
+.btn-apply {
+  width: 10rem;
+  justify-content: center;
+}
+.h4-title {
+  margin-bottom: 0;
+}
+</style>
+<style lang="scss">
+.breadcrumb-start {
+  li:last-child > a {
+    text-decoration: none;
+    font-weight: bold;
+    pointer-events: none;
+    .p-menuitem-text {
+      color: var(--text-color);
+    }
+  }
+
+  .p-menuitem-text {
+    color: $app-link-text;
+  }
+
+  .p-menuitem {
+    color: $app-link-text;
+  }
+}
+</style>
```

Comment:
This should still be scoped. Use the `:deep` annotator to access child classes. If you don't use scope the styles will bleed outside this component.

Reply:
:deep works! Thank you!